lib/cc/analyzer/statsd_container_listener.rb

```diff
@@ -1,9 +1,12 @@
 module CC
   module Analyzer
     class StatsdContainerListener < ContainerListener
-      def initialize(statsd)
+      # rubocop:disable Lint/MissingSuper
+      def initialize(statsd, repo_id: nil)
         @statsd = statsd
+        @repo_id = repo_id
       end
```

Comment:
Millo this feels very forced, can't we just try to read that from the environment instead ?